src/core.cpp

```diff
@@ -12,39 +12,80 @@ void Character::SetMoney(double money) {
 }
 
 
+Action::Action(Character* actor, double score) {
+    this->actor_ = actor;
+    this->score_ = score;
+}
+
 Character* Action::GetActor() {
     return this->actor_;
 }
 
+void Action::SetActor(Character* actor) {
```

Comment:
I generally dislike them because of the verbosity. On the other hand, I avoid even more public member variables on classes, so sometimes I do have a getter and setting -- but for the most part I do other stuff. I use a lot of structs (all fields public and no member functions) and call functions on them.